src/pages/homepage.js

```diff
@@ -0,0 +1,7 @@
+
+function Homepage(){
+
+    return(<div>Homepage</div>);
```

Comment:
I know this is WIP, you can redirect the user to direct products page for now as in Home page we end up looking at footer at top